frontend/src/Components/LoginPage/LoginPage.tsx

```diff
@@ -75,6 +76,9 @@ export const LoginPage = connectToWebSocket(({ socket }) => {
       <a href={TNG_URL} target="_blank" class={classes.logo}>
         <img src={tngLogo} alt={ALT_TNG_LOGO} class={classes.logoImage} />
       </a>
+      <div class={classes.privacyNotice}>
+        <PrivacyNoticeContainer />
```

Comment:
Can we remove the div-wrapper and pass the class to `PrivacyNoticeContainer` direclty?